packages/main/src/Table.js

```diff
@@ -106,6 +106,7 @@ const metadata = {
 		},
 	},
 	events: /** @lends sap.ui.webcomponents.main.Table.prototype */ {
+		popinChange: {},
```

Comment:
Please add docs for the event: 
Event explanation
@ event
@ since 1.0.0-rc.6
@ public

(no spaces after the @)